actioncable/lib/action_cable/channel/base.rb

```diff
@@ -253,7 +253,7 @@ def processable_action?(action)
         def dispatch_action(action, data)
           logger.info action_signature(action, data)
 
-          if method(action).arity == 1
+          if method(action).arity.abs == 1
```

Comment:
Let's flip the condition to `method(action).arity.zero?` instead.

And we'll need a test to ensure this stays fixed.